controllers/tests/utils.py

```diff
@@ -25,7 +26,7 @@ def get_mock_mediator_response_volume(size, name, wwn, array_type, source_id=Non
     volume.pool = "pool1"
     volume.array_type = array_type
     volume.source_id = source_id
-    volume.space_efficiency = space_efficiency
+    volume.space_efficiency = space_efficiency if isinstance(space_efficiency, list) else [space_efficiency]
```

Comment:
I thought this is a `set`, not a `list`

Reply:
missed it, done

Reply:
should be cover as part of CSI-4250 (linked this conversation there)